### PR TITLE
Add OpenInfra Day France

### PR DESCRIPTION
**Pull request by arnaudmorin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -207,6 +207,7 @@ All the data (past and coming) are available publicly in JSON:
...
 * 22: [KCD New York](https://community.cncf.io/events/details/cncf-kcd-new-york-presents-kcd-new-york-2024/) - New York (USA) <a href="https://sessionize.com/kcd-newyork-2024"><img alt="CFP KCD NY" src="https://img.shields.io/static/v1?label=CFP&message=until%2011-February-2024&color=red"></a> <a href="https://forms.cloudnativecommunity.io/s/clreaoxg800509r6a0pct7yng"><img alt="Scholarship" src="https://img.shields.io/static/v1?label=Scholarship&message=until%2021-March-2024&color=yellow" /></a>
 * 22: [TEQnation 2024](https://teqnation.com/) - Utrecht (Netherlands) <a href="https://sessionize.com/teqnation-2024/"><img alt="CFP TEQnation 2024" src="https://img.shields.io/static/v1?label=CFP&message=until%2012-March-2024&color=red"></a>
+* 22: [OpenInfra Day France](https://oideurope2024.openinfra.dev/france/) - Palaiseau (France) <a href="https://oideurope2024.openinfra.dev/france/"><img alt="CFP OpenInfra Day France" src="https://img.shields.io/static/v1?label=CFP&message=until%2012-April-2024&color=green"></a>
```
Comment (scraly) [inline README.md:210]:
```suggestion
* 22: [OpenInfra Day France](https://oideurope2024.openinfra.dev/france/) - Palaiseau (France) <a href="https://openinfrafoundation.formstack.com/forms/2024_openinfra_days_france_cfp"><img alt="CFP OpenInfra Day France" src="https://img.shields.io/static/v1?label=CFP&message=until%2012-April-2024&color=green"></a>
```

**Review by scraly — CHANGES_REQUESTED** [review]
Top
Just a suggestion I've done: in the CFP tag, please give the direct link to the CFP :)

**Comment by arnaudmorin** [comment]
thanks!

**Comment by scraly** [comment]
thx :)

**Pull request merged**